crates/cast/tests/helpers/fixtures.rs

```diff
@@ -18,7 +18,7 @@ use url::Url;
 pub async fn declare_contract(account: &str, path: &str) -> FieldElement {
     let provider = get_provider(URL).expect("Could not get the provider");
     let chain_id = provider.chain_id().await.expect("Could not get chain_id");
-    let account = get_account(
+    let account = get_account_from_accounts_file(
```

Comment:
Currently, `get_account` uses `get_account_from_accounts_file`. There should probably be a separate function `get_account_from_keystore` instead of having the logic in `get_account`